test/integration/parachain/staging/escrow.test.ts

```diff
@@ -122,7 +156,7 @@ describe("escrow", () => {
     }).timeout(500000);
 
     // TODO: Unskip and implement once instant-seal is added to interbtc-standalone. Otherwise this test would take a week.
-    it.skip("should withdraw locked funds", async () => {}).timeout(100000);
+    // it.skip("should withdraw locked funds", async () => {}).timeout(100000);
```

Comment:
Why comment out this line?

Reply:
The empty function was throwing a linting error - I'll uncomment it before merging though, doesn't need to stay commented out.